src/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkImageBrowser.tsx

```diff
@@ -81,7 +81,7 @@ export class LargeArtworkImageBrowser extends React.Component<
                 width="100%"
                 px={hasMultipleImages ? [2, 2, 0] : 0}
                 height="60vh"
-                maxHeight={image.height}
+                maxHeight="3000px"
```

Comment:
Can we add a comment just above this line about the reason this is needed, and perhaps link to JIRA ticket?